lib/id3vx/frame.ex

```diff
@@ -496,6 +519,54 @@ defmodule Id3vx.Frame do
     IO.iodata_to_binary([header, frame_binary])
   end
 
+  def encode_frame(%Frame{id: "COMR"} = frame, %{version: 3} = tag) do
+    %{
+      encoding: encoding,
+      price: price,
+      valid_until: valid_until,
+      contact_url: contact_url,
+      recieved_as: recieved_as,
+      seller_name: seller_name,
+      description: description,
+      picture_mime: picture_mime,
+      logo: logo
+    } = frame.data
+
+    null_byte = get_null_byte(encoding)
+    encoding_byte = get_encoding_byte(encoding)
+    contact_url = convert_string(:iso8859_1, contact_url)
```

Comment:
This conversion should also be applied to:

- price
- valid until
- picture mime